src/design.jl

```diff
@@ -97,6 +97,40 @@ end
 """
 $(TYPEDSIGNATURES)
 
+```jldoctest
+julia> fold!(PlackettBurman(2))
+8×3 DataFrame
+│ Row │ factor1 │ factor2 │ dummy1 │
+│     │ Int64   │ Int64   │ Int64  │
+├─────┼─────────┼─────────┼────────┤
+│ 1   │ 1       │ 1       │ 1      │
+│ 2   │ 1       │ -1      │ -1     │
+│ 3   │ -1      │ -1      │ 1      │
+│ 4   │ -1      │ 1       │ -1     │
+│ 5   │ -1      │ -1      │ -1     │
+│ 6   │ -1      │ 1       │ 1      │
+│ 7   │ 1       │ 1       │ -1     │
+│ 8   │ 1       │ -1      │ 1      │
+
+```
+"""
+function fold!(design::AbstractDesign)
```

Comment:
Maybe this function should be called `mirror!`?

Reply:
I don't think so. Folding is a.f.a.i.k. the general name for this procedure:
- [c.f. e.g. ](https://support.minitab.com/en-us/minitab/18/help-and-how-to/modeling-statistics/doe/supporting-topics/factorial-and-screening-designs/what-is-folding/)
- pyDOE uses the same function name: [pyDOE documentation](https://pythonhosted.org/pyDOE/factorial.html?highlight=fold)